src/views/home.js

```diff
@@ -1,9 +1,27 @@
-import React, {PureComponent} from 'react';
+import React, {Component} from 'react';
+import {compose} from 'redux';
+import {Link} from 'react-router';
+import Panel from 'focus-components/panel';
+import Button from 'focus-components/button';
 
-class Home extends PureComponent {
-    render() {
-        return <h1>Bienvenue sur le tutoriel de Focus v3</h1>
-    }
+const routes = [
+    {route: '/finances/120', destination: 'Finance List', description: 'Exemple d\'un formulaire avec un ListFor', title: 'Finance List'}
+];
+
+const Home = props => {
+    return(
+        <div style={{ display: 'flex', flexWrap: 'wrap', justifyContent: 'center'}}>
```

Comment:
Could you please avoid to use style property ?
We recommand to declare CSS in project instead.